internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -56,7 +56,7 @@ func WithNLB(cidrBlocks []string) func(s *LoadBalancedWebService) {
 // LoadBalancedWebServiceConfig contains fields to configure LoadBalancedWebService.
 type LoadBalancedWebServiceConfig struct {
 	App           *config.Application
-	Env           *config.Environment
+	EnvConfig     *manifest.Environment
```

Comment:
nit maybe:
```suggestion
	EnvManifest     *manifest.Environment
```
`EnvConfig` makes me think of the `config` pkg